lochness/helpers/db.py

```diff
@@ -97,12 +97,13 @@ def get_db_credentials(config_file: Path, db: str = "postgresql") -> Dict[str, s
     db_params = config.parse(path=config_file, section=db)
 
     if "key_file" in db_params:
-        key_file = Path(db_params["key_file"])
+        key_file_str: str = db_params["key_file"]  # type: ignore
```

Comment:
what's the comment `# type: ignore`? Does this command somehow work with your vscode?